code/game/objects/items/devices/traitordevices.dm

```diff
@@ -223,3 +223,85 @@ effective or pretty fucking useless.
 		else
 			charge = min(max_charge,charge + 50) //Charge in the dark
 		animate(user,alpha = Clamp(255 - charge,0,255),time = 10)
+
+
+/* Research Director Syndie teleporter */
+/obj/item/device/experimental_teleporter
```

Comment:
[Please put all new code under the hippie module](https://github.com/HippieStation/HippieStation13/blob/master/code/modules/hippie/README.txt)